src/Modal/index.tsx

```diff
@@ -88,36 +128,48 @@ const Modal: FunctionComponent<Props> = ({
     }
   }
 
+  const headerProps = {
+    variant: header.variant || 'h1',
+    align: header.align || 'left',
+    className: styles['modal-header']
+  }
+
+  const subheaderProps = {
+    variant: subheader.variant || 'h2',
+    align: subheader.align || 'left',
+    className: styles['modal-subheader']
+  }
+
+  const customModalClasses = rest.className ? rest.className + '' : ''
```

Comment:
this isn't necessary. The point of `clsx` is to weed out falsey className values. If it is undefined, clsx won't show it.